packages/cf-component-page/src/PageHeader.js

```diff
@@ -1,20 +1,29 @@
 import React, { PropTypes } from 'react';
+import { createComponent } from 'cf-style-container';
 
-class PageHeader extends React.Component {
-  render() {
-    return (
-      <header className="cf-page__header">
-        <h1 className="cf-page__title">{this.props.title}</h1>
-        {this.props.subtitle &&
-          <p className="cf-page__subtitle">{this.props.subtitle}</p>}
-      </header>
-    );
-  }
-}
+const styles = props => {
+  const theme = props.theme;
+  return {
+    '& h1': {
+      marginBottom: theme.marginBottom
+    },
+    '& p': {
+      marginTop: theme.marginTop,
+      fontSize: theme.fontSize,
+      color: theme.defaultColor
+    }
+  };
+};
+const PageHeader = ({ title, subtitle, className }) => (
+  <header className={className}>
+    <h1 className={className}>{title}</h1>
+    {subtitle && <p className={className}>{subtitle}</p>}
```

Comment:
before your changes header and subtitle had different classNames, now you apply the same. It won't work like you expect.